programs/conditional_vault/src/instructions/resolve_question.rs

```diff
@@ -0,0 +1,33 @@
+use super::*;
+
+#[derive(AnchorSerialize, AnchorDeserialize)]
+pub struct ResolveQuestionArgs {
+    pub payout_numerators: Vec<u32>,
+}
+
+#[derive(Accounts)]
+#[instruction(args: ResolveQuestionArgs)]
+pub struct ResolveQuestion<'info> {
+    #[account(mut, has_one = oracle)]
+    pub question: Account<'info, Question>,
+    pub oracle: Signer<'info>,
+}
+
+impl ResolveQuestion<'_> {
+    pub fn handle(ctx: Context<Self>, args: ResolveQuestionArgs) -> Result<()> {
+        let question = &mut ctx.accounts.question;
+
+        require_eq!(
+            args.payout_numerators.len(),
+            question.num_outcomes(),
+            VaultError::InvalidNumPayoutNumerators
+        );
+
+        question.payout_denominator = args.payout_numerators.iter().sum();
```

Comment:
what's going on here?

Reply:
`payout_numerators` vector is relative share of tokens to be distributed between different answers

sum of all the numerators is the denominator (suppose to guarantee sum to 100%)

theres likely going to be dust / rounding errors from this (and should be handled)

Reply:
ok wow, i need to review your big write up on the structure. That seems hard to bet on. If something can be 70% considered highly effective and 30% considered moderately effective. Like maybe in most cases it should just resolve to a single outcome. But having to balance the idea it might be a mix of outcomes, it's a foreign idea at the moment.

Reply:
this allows you to do it on any metric tho, that's the nice thing. like if you want to lose weight but you're not sure which plan is best, you can create conditional markets on what would be your weight dependant on which plan you eat

Reply:
> `payout_numerators` vector is relative share of tokens to be distributed between different answers
> 
> sum of all the numerators is the denominator (suppose to guarantee sum to 100%)
> 
> theres likely going to be dust / rounding errors from this (and should be handled)

we're always rounding in the vault's favor, so there will be dust left in the vault and that's okay